pkg/3scale/amp/operator/apicast_options_provider.go

```diff
@@ -34,11 +34,6 @@ func NewApicastOptionsProvider(apimanager *appsv1alpha1.APIManager, client clien
 }
 
 func (a *ApicastOptionsProvider) GetApicastOptions() (*component.ApicastOptions, error) {
-	imageOpts, err := NewAmpImagesOptionsProvider(a.apimanager).GetAmpImagesOptions()
```

Comment:
Do you think we might actually need this for https://issues.redhat.com/browse/THREESCALE-7481 ?

Reply:
:+1: